fedora_messaging/tests/unit/test_cli.py

```diff
@@ -174,32 +170,19 @@ def test_wrong_cli_bindings(self, mock_consume, mock_importlib):
             ],
         )
         mock_importlib.import_module.called_once_with("mod")
-        mock_consume.assert_not_called()
-        self.assertIn(
-            "You must define all three of exchange, queue_name and"
-            " routing_key, or none of them to use the configuration",
-            result.output,
-        )
-        self.assertEqual(1, result.exit_code)
-
-    @mock.patch.dict(
-        "fedora_messaging.config.conf", {"bindings": None, "callback": "mod:callable"}
-    )
-    @mock.patch("fedora_messaging.cli.importlib")
-    @mock.patch("fedora_messaging.cli.api.consume")
-    def test_missing_cli_and_conf_bindings(self, mock_consume, mock_importlib):
```

Comment:
The diff is a bit confusing here, but I got rid of this test because it tests a scenario that can't happen - when the configuration is loaded, the bindings and queues are checked to make sure they're valid (and not None since the default isn't)